packages/toolkit/src/query/core/buildThunks.ts

```diff
@@ -313,6 +313,15 @@ export function buildThunks<
       if (error instanceof HandledError) {
         return rejectWithValue(error.value, { baseQueryMeta: error.meta })
       }
+      if (
+        typeof process !== 'undefined' &&
+        process.env.NODE_ENV === 'development'
+      ) {
+        console.log(`An unhandled error occured processing a request for the endpoint ${arg.endpointName}.
```

Comment:
```suggestion
        console.warn(`An unhandled error occured processing a request for the endpoint ${arg.endpointName}.
```

Reply:
Actually I think both of these should be `console.error`. Dunno what I was (not) thinking there.

Reply:
You put the intention in the description though! 🏆